website/views.py

```diff
@@ -133,57 +131,17 @@ def index():
     try:  # Check if we're on an institution landing page
         #TODO : make this way more robust
         institution = Institution.objects.get(domains__contains=[request.host.lower()], is_deleted=False)
-        inst_dict = serialize_institution(institution)
-        inst_dict.update({
-            'home': False,
-            'institution': True,
-            'redirect_url': '{}institutions/{}/'.format(DOMAIN, institution._id),
-        })
-
-        return inst_dict
+        return redirect('{}institutions/{}/'.format(DOMAIN, institution._id))
```

Comment:
If on institution landing page, navigating to index route should still put you back on institution landing page.